src/lib/protocols/presentproof/PresentationStates.ts

```diff
@@ -0,0 +1,10 @@
+export enum PresentationStates {
+    PROPOSAL_SENT = "proposal_sent",
+    PROPOSAL_RECEIVED = "proposal_received",
+    REQUEST_SENT = "request_sent",
+    REQUEST_RECEIVED = "request_received",
+    PRESENTATION_SENT = "presentation_sent",
+    PRESENTATION_RECEIVED = "presentation_received",
+    VERIFIED = "verified",
+    PRESENTATION_ACKED = "presentation_acked"
+}
```

Comment:
We're using [TypeScript String enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) with the following convention:

```
export enum ConnectionState {
  Init = 'INIT',
  Invited = 'INVITED',
  Requested = 'REQUESTED',
  Responded = 'RESPONDED',
  Complete = 'COMPLETE',
}
```

Reply:
Sure, will update. Thanks